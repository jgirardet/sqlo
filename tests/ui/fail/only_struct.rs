#[derive(sqlo::Sqlo)]
enum NoEnumStruct {}

#[derive(sqlo::Sqlo)]
struct NoTuppleStruct(String);

#[derive(sqlo::Sqlo)]
struct NoUnitStruct(String);

fn main() {}
