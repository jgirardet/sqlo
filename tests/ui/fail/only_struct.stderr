error: Unsupported shape `enum`
 --> tests/ui/fail/only_struct.rs:1:10
  |
1 | #[derive(sqlo::Sqlo)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `sqlo::Sqlo` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Unsupported shape `newtype`
 --> tests/ui/fail/only_struct.rs:4:10
  |
4 | #[derive(sqlo::Sqlo)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `sqlo::Sqlo` (in Nightly builds, run with -Z macro-backtrace for more info)

error: Unsupported shape `newtype`
 --> tests/ui/fail/only_struct.rs:7:10
  |
7 | #[derive(sqlo::Sqlo)]
  |          ^^^^^^^^^^
  |
  = note: this error originates in the derive macro `sqlo::Sqlo` (in Nightly builds, run with -Z macro-backtrace for more info)
